### PR TITLE
[Feature] [57824] Initial Project

### PR DESCRIPTION
**Pull request by DoQuangKhanh0209**
## [Related Tickets]
[https://edu-redmine.sun-asterisk.vn/issues/57825](url)
### [What]
- Initial Project
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/Zts4P8 |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Message | Sun* commit message | Template: [TASK/FREATURE/BUG/SUPPORT][RedmineID] + message |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: Making-Cocktails/Making-Cocktails/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        let container = NSPersistentContainer(name: "Making_Cocktails")
+        container.loadPersistentStores(completionHandler: { (storeDescription, error) in
+            if let error = error as NSError? {
+                fatalError("Unresolved error \(error), \(error.userInfo)")
+            }
+        })
+        return container
+    }()
+
+    func saveContext () {
```
Comment (tunghs-1253) [inline Making-Cocktails/Making-Cocktails/AppDelegate.swift]: Không define method coreData trong AppDelegate anh

File: Making-Cocktails/Making-Cocktails/Base.lproj/LaunchScreen.storyboard [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tunghs-1253) [inline Making-Cocktails/Making-Cocktails/Base.lproj/LaunchScreen.storyboard:1]: Sao init chỉ create empty project mà a đã làm launchScreen ạ

File: Making-Cocktails/Making-Cocktails/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,18 @@
+//
+//  ViewController.swift
+//  Making-Cocktails
+//
+//  Created by Quang Khanh on 14/12/2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (tunghs-1253) [inline Making-Cocktails/Making-Cocktails/ViewController.swift]:
```suggestion
final class ViewController: UIViewController {
```

File: Making-Cocktails/Making-Cocktails/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
```
Comment (tunghs-1253) [inline Making-Cocktails/Making-Cocktails/ViewController.swift]: Xoá dòng trống

File: Making-Cocktails/Making-Cocktails/AppDelegate.swift (hunk 1 above)
Comment (DoQuangKhanh0209) [reply Making-Cocktails/Making-Cocktails/AppDelegate.swift]: @tunghs-1253 
không phải a define đâu mà lúc tạo coreData nó có như thế á

File: .gitignore [hunk 5]
```diff
@@ -87,4 +87,4 @@ fastlane/test_output
 # After new code Injection tools there's a generated folder /iOSInjectionProject
 # https://github.com/johnno1962/injectionforxcode
```
Comment (tunghs-1253) [inline .gitignore:88]: Cái này là cái gì thế anh

**Pull request merged**